docs/api/core/cell.md

```diff
@@ -2,7 +2,7 @@
 title: Cell
 ---
 
-These are **core** options and API properties for all cells. More options and API properties are available for other [table features](../../guide/features.md).
+These are **core** options and API properties for all cells. More options and API properties are available for other [table features](../../guide/features).
```

Comment:
When clicking on the `table features`, the page is redirected to the home page instead of the table features page. This change is to make the page redirect properly.